recipes/brotli/all/conanfile.py

```diff
@@ -55,18 +67,29 @@ def package(self):
         cmake.install()
         tools.rmdir(os.path.join(self.package_folder, "lib", "pkgconfig"))
 
-    def _get_libraries(self, shared):
-        libs = ["brotlienc", "brotlidec", "brotlicommon"]
-        if not shared:
-            libs = ["{}-static".format(l) for l in libs]
-        return libs
-
     def package_info(self):
-        self.cpp_info.names["cmake_find_package"] = "Brotli"
-        self.cpp_info.names["cmake_find_package_multi"] = "Brotli"
```

Comment:
Changing this breaks existing consumers. How is the expected `FindBrotli.cmake` filename?

Reply:
There are no official module and config files for Brotli.

Reply:
I can find it associated to curl dependency in the CMake repo, I know those are not the official modules, but... why changing what it is written now and someone might be using if there is no _strong_ reason for it?  

https://gitlab.kitware.com/cmake/cmake/-/tree/master/Utilities/cmcurl/CMake

Reply:
Yes I can restore those lines. It's just I think it's bad practice that conan recipes suggest unofficial CMake names.

Reply:
I agree with @SpaceIm.
But I vote for keeping it.
Let's hope brotli follows this naming scheme when they, if ever, export cmake scripts.

Reply:
Same here, but as it is already written... maybe CCI starts to be the single source of truth in some near future 😺